nengo/builder/connection.py

```diff
@@ -76,7 +76,11 @@ def get_targets(conn, eval_points):
     else:
         targets = np.zeros((len(eval_points), conn.size_mid))
         for i, ep in enumerate(eval_points[:, conn.pre_slice]):
-            targets[i] = conn.function(ep)
+            out = conn.function(ep)
+            if out is None:
+                raise BuildError("Building %s: Connection function returned "
```

Comment:
The error message is good! I am just wondering if those memory locations (and other meta-data about Ensembles) is really necessary, e.g.,:
```BuildError: Building <Connection from <Ensemble (unlabeled) at 0x7fe738f619d0> to <Ensemble (unlabeled) at 0x7fe738f61b90> computing 'f'>: Connection function returned None. Cannot solve for decoders.```

I find they obfuscate the message, but if this is standard in error reporting for Nengo, then it's probably better to leave it as it is.